client/actions/businessActions.js

```diff
@@ -162,12 +138,12 @@ export function allBusinesses(businesses) {
 
 /**
  * @description - Gets all businesses
- *
+ * @param {*} pageNum
```

Comment:
Jerry I like what you did here, pretty nice implementation on the pagination. A quick one on your naming of variables, please kindly complete the "pageNum" to "pageNumber".